src/main/resources/modules/hospice_treatment.json

```diff
@@ -0,0 +1,281 @@
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "310417005",
+          "display": "Certification procedure (procedure)"
+        }
+      ],
+      "direct_transition": "Notifications"
+    },
+    "Notifications": {
+      "type": "Procedure",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "185087000",
+          "display": "Notifications (procedure)"
+        }
+      ],
+      "direct_transition": "Initial_Patient_Assessment"
+    },
+    "Initial_Patient_Assessment": {
+      "type": "Procedure",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "315639002",
+          "display": "Initial patient assessment (procedure)"
+        }
+      ],
+      "direct_transition": "Individualized_Care_Plan"
+    },
+    "Individualized_Care_Plan": {
+      "type": "Procedure",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "370789001",
+          "display": "Development of individualized plan of care (procedure)"
+        }
+      ],
+      "direct_transition": "Begin_Day"
+    },
+    "Begin_Day": {
+      "type": "Counter",
+      "attribute": "hospice_days",
+      "action": "decrement",
+      "amount": 1,
+      "direct_transition": "Hospice_Care"
+    },
+    "Hospice_Care": {
+      "type": "Procedure",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "385763009",
+          "display": "Hospice care (regime/therapy)"
+        }
+      ],
+      "range": {
+        "high": 60,
+        "low": 30,
+        "unit": "minutes"
+      },
+      "direct_transition": "End_Day"
+    },
+    "End_Day": {
+      "type": "Delay",
+      "exact": {
+        "quantity": 1,
+        "unit": "days"
+      },
+      "conditional_transition": [
+        {
+          "transition": "Alive Check",
+          "condition": {
+            "condition_type": "Attribute",
+            "attribute": "hospice_days",
+            "operator": "<",
+            "value": 0
+          }
+        },
+        {
+          "transition": "Begin_Day"
+        }
+      ]
+    },
+    "Patient_Discharge": {
+      "type": "Procedure",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "58000006",
+          "display": "Patient discharge (procedure)"
+        }
+      ],
+      "direct_transition": "Discharged Alive"
+    },
+    "Died_in_Hospice": {
+      "type": "ConditionOnset",
+      "assign_to_attribute": "",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "876882001",
+          "display": "Died in hospice (finding)"
+        }
+      ],
+      "direct_transition": "Discharged Dead"
+    },
+    "Alive Check": {
+      "type": "Simple",
+      "conditional_transition": [
+        {
+          "transition": "Died_in_Hospice",
+          "condition": {
+            "condition_type": "Attribute",
+            "attribute": "days_until_death",
```

Comment:
Are we sure it's possible to go down this transition? Because the simulation stops when a patient dies, I wouldn't expect the Lifecycle module to ever run and set `days_until_death` < 0 when the patient is dead .

Reply:
Honestly, I wasn't sure if it would ever get executed or not. My suspicion is that it does not get executed, or if it does, rarely. So, I left it in just in case this module and those states are executed before the death time. In other words, it is the day they die (`=0`), but before the hour/minute in which the death time was set at.

Reply:
Leaving as-is.